src/script/builder/tir/for_frame.cc

```diff
@@ -92,6 +94,43 @@ ForFrame Grid(Array<PrimExpr> extents) {
 
 TVM_REGISTER_NODE_TYPE(ForFrameNode);
 
+TVM_REGISTER_GLOBAL("script.builder.tir.EnterForFrame")
+  .set_body_method<ForFrame>(&ForFrameNode::EnterWithScope);
+
+TVM_REGISTER_GLOBAL("script.builder.tir.ExitForFrame")
+  .set_body_method<ForFrame>(&ForFrameNode::ExitWithScope);
```

Comment:
I suppose there is no need to expose these two methods, because `FrameNode::EnterWithScope/ExitWithScope` are both exposed already. Is my understanding correct?

Reply:
Well, the `__enter__` and `__exit__` has no inheritance call. If we do not expose `ForFrameNode::EnterWithScope/ExitWithScope`, we have to call `super().__enter__` and `super().__exit__` in `ForFrameNode`.
In addition, the exposure of `ForFrameNode::EnterWithScope` is necessary. Since we support statement like `with T.Grid(128, 128) as (i, j)`, we should have specific `ForFrame.__enter__`, different from `Frame.__enter__`.